doc/rose-rug-advanced-tutorials-macro.html

```diff
@@ -589,6 +589,21 @@ <h3 class="alwayshidden">Example Metadata Option</h3>
             </div>
 
             <div class="slide">
+              <h2 id="reporter-macros">Reporter Macros</h2>
+              <p>There are also reporter macros which can be used
+              where you need to output some information about a
+              configuration. A reporter macro takes the same form
+              as validator and transform macros but does not require
+              a return value.</p>
+
+              <pre class="prettyprint">
+    def report(self, config, meta_config=None):
```

Comment:
Once that change is made I'll work through the tutorial and make sure everything's fine with Rose edit integration etc as that isn't checked in the test battery.